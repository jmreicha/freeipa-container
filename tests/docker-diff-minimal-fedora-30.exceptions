# Workaround 1611623
^C /var/log/[bw]tmp$
# This is new in Fedora rawhide (30)
^C /var$
^C /var/tmp$
^[AD] /var/tmp/systemd-private-.*-dbus-broker.service-
