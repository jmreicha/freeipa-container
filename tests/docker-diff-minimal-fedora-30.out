A /lextab.py
A /yacctab.py
C /etc/machine-id
